client/src/pages/private/ParticipantTable.js

```diff
@@ -136,9 +136,11 @@ export default () => {
   const { auth } = AuthContext.useAuth();
   const roles = useMemo(() => auth.user?.roles || [], [auth.user?.roles]);
   const sites = useMemo(() => auth.user?.sites || [], [auth.user?.sites]);
+  let hasWithdrawnParticipant = false;
```

Comment:
Should this be state?

Reply:
This not used inside any view / jsx code, so kept it as simple variable.

Reply:
cool!